example-library/src/test/java/uk/gov/gchq/palisade/example/rule/BankDetailsRulesTest.java

```diff
@@ -16,92 +16,90 @@
 
 package uk.gov.gchq.palisade.example.rule;
 
+import org.junit.Before;
 import org.junit.Test;
 
 import uk.gov.gchq.palisade.Context;
 import uk.gov.gchq.palisade.User;
 import uk.gov.gchq.palisade.example.common.ExampleUser;
 import uk.gov.gchq.palisade.example.common.Purpose;
-import uk.gov.gchq.palisade.example.common.Role;
 import uk.gov.gchq.palisade.example.common.TrainingCourse;
-import uk.gov.gchq.palisade.example.hrdatagenerator.types.BankDetails;
 import uk.gov.gchq.palisade.example.hrdatagenerator.types.Employee;
-import uk.gov.gchq.palisade.jsonserialisation.JSONSerialiser;
 
 import java.util.Random;
 
-import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
 import static org.junit.Assert.assertNull;
-import static org.junit.Assert.assertTrue;
 
 public class BankDetailsRulesTest {
 
-    private static final Employee TEST_EMPLOYEE = Employee.generate(new Random(1));
-    private static final User TEST_USER_NOT_PAYROLL = new User().roles("Not Payroll").userId("UserId"); // Role not in Payroll
-    private static final User TEST_USER_PAYROLL = new User().roles(Role.PAYROLL.name()).userId("UserId"); // Role in Payroll
+    private static final User EXAMPLE_USER_WITHOUT_PAYROLL = new ExampleUser().userId("UserId"); // Role not in Payroll
+    private static final User EXAMPLE_USER_WITH_PAYROLL = new ExampleUser().trainingCompleted(TrainingCourse.PAYROLL_TRAINING_COURSE).userId("UserId"); // Role in Payroll
+    private static final User STANDARD_USER = new User().userId("UserId"); // Role not in Payroll
```

Comment:
```suggestion
    private static final User EXAMPLE_USER_WITHOUT_PAYROLL = new ExampleUser().userId("UserId");
    private static final User EXAMPLE_USER_WITH_PAYROLL = new ExampleUser().trainingCompleted(TrainingCourse.PAYROLL_TRAINING_COURSE).userId("UserId");
    private static final User STANDARD_USER = new User().userId("UserId");
```
The comments are no longer relevant as the PAYROLL role has been deleted.